akka-docs/src/test/scala/docs/stream/operators/Map.scala

```diff
@@ -0,0 +1,17 @@
+package docs.stream.operators
+
+//#imports
+import akka.NotUsed
+import akka.stream.scaladsl._
+
+//#imports
+
+object Map {
+
+  //#map
+  def mapExample: Source[String, NotUsed] = {
+    val source: Source[Int, NotUsed] = Source(1 to 10)
+    source.map(elem => elem.toString)
```

Comment:
Wrapping it in a function might be confusing to new users... perhaps we could assign it like `val mapped: Source[String, NotUsed] = source.map(elem => elem.toString)`?

Reply:
You are right. Changed the example.